docs/configuration/prefill/stuf_bg.rst

```diff
@@ -0,0 +1,26 @@
+.. _configuration_prefill_stuf_bg:
+
+=======
+StUF-BG
+=======
+
+StUF-BG (StUF Basis Gegegevens) is a message standard that allows retrieving of
+personal information by API from municipalities and government organisations.
+Open Forms can be configured to use this service to prefill personal data of
+the person filling out the form.
+
+1. You will need to have a contract with a municipality or government organisation to use this plugin.
```

Comment:
Is this correct?  Is there a specific organisation they would need a contract with?